### DISCUSSION TITLE
Add test docs in the `README.md` file

### DISCUSSION THREAD

**Discussion by luislucena16** [PaulRBerg/prb-proxy#188] [Ideas]
I think it would be ideal add how to run tests in `README.md` file, an ideal example would be to see when a `.delegetacall` is executed or when it fails due to `collision`

**Comment by PaulRBerg** [comment]
Hey! What do you mean by "add test runs"?

**Reply by luislucena16** [reply]
hey @PaulRBerg  sorry, it was in relation to how to run the tests, I've changed it!

**Reply by PaulRBerg** [reply]
Got it.

PRs welcome!

**Reply by luislucena16** [reply]
understood! the repository also needs a contribution guide! i will do these jobs separately!
